config/logging.yml

```diff
@@ -0,0 +1,29 @@
+version: 1
+formatters:
+  simple:
+    format: '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: DEBUG
+    formatter: simple
+    stream: ext://sys.stdout
+  file:
+    class : logging.handlers.RotatingFileHandler
+    formatter: simple
+    filename: /tmp/prom2teams.log
+    maxBytes: 2097152 #200MB
```

Comment:
7 files * 200 MB per file: 1.4 GB 

Probably is too much

Reply:
updated